src/node.cc

```diff
@@ -2703,90 +2720,162 @@ struct DLib {
     uv_dlclose(&lib_);
   }
 #endif  // !__POSIX__
+
+  DLib(const DLib& other) = delete;
+  DLib(DLib&& other) = delete;
+  DLib& operator=(const DLib& other) = delete;
+  DLib& operator=(DLib&& other) = delete;
+
+  void AddEnvironment(Environment* env) {
+    if (users_.count(env) > 0) return;
+    users_.insert(env);
```

Comment:
Not sure if this won't hurt readability, but these two lines can be merged into something like `if (!users_.insert(env).second) return;`.

Reply:
Hm yeah, I think the current way is a bit more readable …